SeaPublicWebsite/ExternalServices/EpbEpc/EpbEpcConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SeaPublicWebsite.ExternalServices.EpbEpc
+{
+    public class EpbEpcConfiguration
+    {
+        public const string ConfigSection = "EpbEpc";
+        
+        public string BaseUrl { get; set; }
+        public string Username { get; set; }
+        public string Password { get; set; }
+    }
+}
```

Comment:
I assume there needs to be a similar file BreConfiguration, happy to add it myself later :)

Reply:
Yes, exactly :)